easybuild/easyconfigs/f/fastp/fastp-0.20.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,33 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
```

Comment:
Why the switch to `MakeCp`, the existing easyconfigs use `ConfigureMake` (which makes sense since you can pass the installation prefix to use to `make` via `PREFIX=...`